### PR TITLE
feat: cli can take custom alphabet as args

### PR DESCRIPTION
**Pull request by Phaired**
_(Maintainer's note - this is a counterpart to https://github.com/robertknight/ocrs/pull/100 for the CLI tool)_

### CODE REVIEW

File: ocrs-cli/src/main.rs [hunk 1]
```diff
@@ -182,6 +190,10 @@ Options:
 
     Use a custom text recognition model
 
+  -a, --alphabet \"alphabet\"
+
+    Use a custom alphabet for recognition
```
Comment (robertknight) [inline ocrs-cli/src/main.rs:195]:
```suggestion
    Specify the alphabet used by the recognition model.
```

I think it is important to clarify here that the alphabet needs to match what the recognition model was trained with.

File: ocrs-cli/src/main.rs [hunk 2]
```diff
@@ -233,6 +245,7 @@ Advanced options:
         text_map,
         text_mask,
         text_line_images,
+        alphabet,
```
Comment (robertknight) [inline ocrs-cli/src/main.rs:248]: Can you keep these fields sorted.

File: ocrs-cli/src/main.rs [hunk 3]
```diff
@@ -104,6 +104,10 @@ struct Args {
 
     /// Extract each text line found and save as a PNG image.
     text_line_images: bool,
+
+    /// Custom alphabet for recognition.
```
Comment (robertknight) [inline ocrs-cli/src/main.rs:108]:
```suggestion
    /// Alphabet used by the recognition model.
```

**Review by robertknight — CHANGES_REQUESTED** [review]
Thanks for the patch. I requested some adjustment to the wording to make it clear that the alphabet has to match the model. The user can't just use whatever alphabet they like.

In future I think it will make sense to have a way to embed the alphabet in the model file, or provide it as part of a configuration file that is shipped alongside the model. In the interim, adding a CLI argument is useful.

**Comment by Phaired** [comment]
I agree that embedding the alphabet in the model file would be a good feature.
Sorry for the many commits.

**Comment by robertknight** [comment]
> Sorry for the many commits.

That's not a problem. As a maintainer, I can also do this when merging a PR using the ["Squash and merge"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges#squash-and-merge-your-commits) option.

For future reference, you can use [`git rebase --interactive`](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase) followed by `git push --force` to tidy up the commit history by editing commit messages, merging commits or other tasks.

**Review by robertknight — APPROVED** [review]

**Pull request merged**